.github/workflows/publishing.yml

```diff
@@ -46,10 +48,20 @@ jobs:
               done
               rm -rf build_info build_info.zip
             fi
+            if [ "${name#pycudaq-}" != "$name" ]; then
```

Comment:
For my understanding, this is going to return a wheel for each python version 3.9-3.11?

Reply:
3.8-3.11, yes (i.e. for every version listed in the deployment workflow).